service/src/test/java/bio/terra/pipelines/controller/PipelinesApiControllerTest.java

```diff
@@ -98,6 +104,55 @@ void getPipelineOk() throws Exception {
             .readValue(result.getResponse().getContentAsString(), ApiPipelineWithDetails.class);
 
     assertEquals(pipelineName, response.getPipelineName());
+    assertEquals(pipelineName, response.getPipelineName());
```

Comment:
is this a duplicate of the previous line?

Reply:
😬